network-api/networkapi/wagtailpages/pagemodels/customblocks/typeform_block.py

```diff
@@ -0,0 +1,24 @@
+from wagtail.core import blocks
+
+
+class TypeformBlock(blocks.StructBlock):
+    url = blocks.URLBlock(
+        help_text="The URL of the published Typeform"
+    )
+
+    button_type = blocks.ChoiceBlock(
+        choices=[
+            ('btn-primary', 'Primary button'),
+            ('btn-secondary', 'Secondary button'),
+        ],
+        default='btn-primary',
+    )
+
+    button_text = blocks.CharBlock(
+        required=True
+    )
+
+    class Meta:
```

Comment:
As per https://stackoverflow.com/a/43561880/740553:

```suggestion
    @property
    def media(self):
        return forms.Media(
            js=["https://embed.typeform.com/embed.js"],
        )

    class Meta:
```